src/meta/proto-conv/tests/it/proto_conv.rs

```diff
@@ -31,6 +31,23 @@ fn s(ss: impl ToString) -> String {
     ss.to_string()
 }
 
+fn new_db_meta_v2() -> mt::DatabaseMeta {
+    mt::DatabaseMeta {
+        engine: "44".to_string(),
+        engine_options: btreemap! {s("abc") => s("def")},
+        options: btreemap! {s("xyz") => s("foo")},
+        created_on: Utc.ymd(2014, 11, 28).and_hms(12, 0, 9),
+        updated_on: Utc.ymd(2014, 11, 29).and_hms(12, 0, 9),
+        comment: "foo bar".to_string(),
+        drop_on: None,
+        shared_by: BTreeSet::new(),
+        from_share: Some(share::ShareNameIdent {
+            tenant: "tenant".to_string(),
+            share_name: "share".to_string(),
+        }),
+    }
+}
+
```

Comment:
This function will build a message of the current version, not `v2`, IMHO.

Maybe it'd better qualify it with `_from_share()` instead of `_v2()`.